src/qt/forms/overviewpage.ui

```diff
@@ -394,23 +414,31 @@
               </property>
              </widget>
             </item>
-            <item row="6" column="1">
-             <widget class="QLabel" name="labelCpid">
+            <item row="8" column="0">
+             <widget class="QLabel" name="label">
               <property name="text">
-               <string notr="true"/>
+               <string>Est. TTS:</string>
               </property>
-              <property name="textInteractionFlags">
-               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
+             </widget>
+            </item>
+            <item row="8" column="1">
+             <widget class="QLabel" name="labelETTS">
+              <property name="text">
+               <string/>
               </property>
              </widget>
             </item>
-            <item row="7" column="1">
-             <widget class="QLabel" name="labelStatus">
+            <item row="9" column="0">
+             <widget class="QLabel" name="label_4">
               <property name="text">
-               <string notr="true"/>
+               <string>Est. RR/day:</string>
               </property>
-              <property name="textInteractionFlags">
-               <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
+             </widget>
+            </item>
+            <item row="9" column="1">
+             <widget class="QLabel" name="labelERRperday">
+              <property name="text">
+               <string/>
```

Comment:
You are renaming some labels to generic names like "label_4" and give proper names to others. Why?

Reply:
I did this through the designer. It does that automatically.

Reply:
This code in *.ui is generated by the qt designer...

Reply:
This can be modified outside of designer

Reply:
Also the fields with proper names are the ones where information is plugged in via the reference to GlobalStatusStruct by the label pointers in iu.cpp.